plugins/emc/emc_vnx_file_

```diff
@@ -0,0 +1,516 @@
+        echo "${server}_stor_write.value ${graphs[5]}"
+
+        member_elements=$($SSH server_stats $server -monitor kernel.memory -count 1 -terminationsummary no -titles never | grep '^[^[:space:]]')
+        IFS=$' ' read -ra graphs <<< "$member_elements"
+
+        echo -e "\nmultigraph emc_vnx_memory"
+        echo "${server}_total.value ${graphs[14]}"
+        echo "${server}_used.value ${graphs[15]}"
+        echo "${server}_free.value ${graphs[12]}"
+        echo "${server}_freebuffer.value ${graphs[1]}"
+        echo "${server}_encumbered.value ${graphs[8]}"
+
+        echo -e "\nmultigraph emc_vnx_filecache"
+        echo "${server}_highw_hits.value ${graphs[2]}"
+        echo "${server}_loww_hits.value ${graphs[6]}"
+        echo "${server}_w_hits.value ${graphs[7]}"
+        echo "${server}_hits.value ${graphs[4]}"
+	echo "${server}_lookups.value ${graphs[5]}"
+        
+        echo -e "\nmultigraph emc_vnx_fileresolve"
+        echo "${server}_dropped.value ${graphs[9]}"
+        echo "${server}_max.value ${graphs[10]}"
+        echo "${server}_used.value ${graphs[11]}"
+
+
+	fi
+	if [ $STATSTYPE = "NFS" ] ; then
+#nfs.v3.op data
+	member_elements=$($SSH server_stats $server -monitor nfs.v3.op -count 1 -terminationsummary no -titles never | sed -ne 's/^.*v3/v3/p')
+	NUMCOL=5
+	LINES=$(wc -l <<< "$member_elements")
+	while IFS=$'\n' read -ra graphs ; do
+		element+=( $graphs )
+	done <<< "$member_elements"
+
+	echo "multigraph vnx_emc_v3_calls_s"
+	for ((i=0; i<$((LINES)); i++ )); do
+		echo "${server}_${element[i*$NUMCOL]}".value "${element[i*$NUMCOL+1]}"
+	done
+
+	echo -e "\nmultigraph vnx_emc_v3_usec_call"
+	for ((i=0; i<$((LINES)); i++ )); do
+		echo "${server}_${element[i*$NUMCOL]}".value "${element[i*$NUMCOL+3]}"
+	done
+
+	echo -e "\nmultigraph vnx_emc_v3_op_percent"
+	for ((i=0; i<$((LINES)); i++ )); do
+		echo "${server}_${element[i*$NUMCOL]}".value "${element[i*$NUMCOL+4]}"
+	done
+
+	element=()
+
+#nfs.v4.op data
+	member_elements=$($SSH server_stats $server -monitor nfs.v4.op -count 1 -terminationsummary no -titles never | sed -ne 's/^.*v4/v4/p')
+	NUMCOL=5
+	LINES=$(wc -l <<< "$member_elements")
+	while IFS=$'\n' read -ra graphs ; do
+		element+=( $graphs )
+	done <<< "$member_elements"
+
+	echo -e "\nmultigraph vnx_emc_v4_calls_s"
+	for ((i=0; i<$((LINES)); i++ )); do
+		echo "${server}_${element[i*$NUMCOL]}".value "${element[i*$NUMCOL+1]}"
+	done
+
+	echo -e "\nmultigraph vnx_emc_v4_usec_call"
+	for ((i=0; i<$((LINES)); i++ )); do
+		echo "${server}_${element[i*$NUMCOL]}".value "${element[i*$NUMCOL+3]}"
+	done
+
+	echo -e "\nmultigraph vnx_emc_v4_op_percent"
+	for ((i=0; i<$((LINES)); i++ )); do
+		echo "${server}_${element[i*$NUMCOL]}".value "${element[i*$NUMCOL+4]}"
+	done
+
+	element=()
+
+#nfs.client data
+	echo -e "\nmultigraph vnx_emc_nfs_client_ops_s"
+        member_elements=$($SSH server_stats server_2 -monitor nfs.client -count 1 -terminationsummary no -titles never | sed -ne 's/^.*id=//p')
+	NUMCOL=9
```

Comment:
It looks like you are mixing tabs and spaces for indentation in the lines above and below.